src/templates/osmosis.yml

```diff
@@ -0,0 +1,23 @@
+networks:
+  mainnet:
+    nodeURL: https://rpc.osmosis.zone/
+    tokenListType: URL
+    tokenListSource: https://raw.githubusercontent.com/osmosis-labs/assetlists/main/osmosis-1/osmosis-1.assetlist.json
```

Comment:
I strongly suggest copying this locally into the lists folder and using the `FILE` list type - see other chains/DEXs for examples. This prevents a whole class of network-related issues that affect users in production.